src/policy_evaluator.rs

```diff
@@ -110,27 +111,22 @@ impl fmt::Debug for PolicyEvaluator {
 }
 
 impl PolicyEvaluator {
-    pub fn from_file(
-        id: String,
-        policy_file: &Path,
-        policy_execution_mode: PolicyExecutionMode,
-        settings: Option<serde_json::Map<String, serde_json::Value>>,
-    ) -> Result<PolicyEvaluator> {
-        PolicyEvaluator::from_contents(id, fs::read(policy_file)?, policy_execution_mode, settings)
-    }
-
-    pub fn from_contents(
+    /// This method should not be used directly. Please use
+    /// `PolicyEvaluatorBuilder` instead.
```

Comment:
We can remove this comment given it's not part of public API

Reply:
I left if on purpose for us, users of the internal `pub(crate)` method